app/nlg/country.py

```diff
@@ -983,27 +992,28 @@ def render(cnc, lexeme, entity):
 		phr = mkPhr(mkUtt(w.ExtAdvS(adv,mkS(mkCl(mkNP(lexeme), mkVP(passiveVP(mkVPSlash(w.rank_2_V2)),mkAdv(w.as_Prep,quality)))))), fullStopPunct)
 		yield " " + cnc.linearize(phr)
 
-
 	pol = positivePol
-	freedom = get_items("P1552",entity,qual=False)
-	if freedom:
-		for quality in freedom:
-			if quality == "Q3174312":
-				quality = mkNP(aSg_Det,mkCN(w.free_1_A,w.country_1_N))
-				break
-			elif quality == "Q47185282":
-				quality = mkNP(aSg_Det,mkCN(w.free_1_A,w.country_1_N))
-				pol     = negativePol
-				break
-			elif quality == "Q47185145":
-				quality = mkNP(aSg_Det,mkCN(mkAP(w.partly_AdA,w.free_1_A),w.country_1_N))
-				break
-			elif quality == "Q7174":
-				quality = mkNP(aSg_Det,mkCN(w.democratic_1_A,w.country_1_N))
-				break
-			quality = None
-		phr = mkPhr(mkUtt(mkS(pol,mkCl(mkNP(w.freedom_in_the_world_PN), mkVP(mkVPSlash(w.consider_6_V3,mkNP(w.it_Pron)), quality)))), fullStopPunct)
-		yield " "+cnc.linearize(phr)
+	for quality in get_items("P1552",entity,qual=False):
+		if quality == "Q3174312":
+			quality = mkNP(aSg_Det,mkCN(w.free_1_A,w.country_1_N))
+			break
+		elif quality == "Q47185282":
+			quality = mkNP(aSg_Det,mkCN(w.free_1_A,w.country_1_N))
+			pol     = negativePol
+			break
+		elif quality == "Q47185145":
+			quality = mkNP(aSg_Det,mkCN(mkAP(w.partly_AdA,w.free_1_A),w.country_1_N))
+			break
+		elif quality == "Q7174":
+			quality = mkNP(aSg_Det,mkCN(w.democratic_1_A,w.country_1_N))
+			break
+		quality = None
+	if cnc.name in ["ParseRus"]: # (VP + accusative) + instrumental
+		phr = mkPhr(mkUtt(mkS(pol,mkCl(mkNP(w.freedom_in_the_world_PN), mkVP(w.Slash3V3(w.consider_6_V3,quality), mkNP(w.it_Pron))))), fullStopPunct)
```

Comment:
I though that I pushed a patch for that but apparently I had only done this locally. The best way to write this is like this: mkVP(w.consider_6_V3,mkNP(w.it_Pron),quality). Then it works for all languages. You need to update the gf-wordnet Python package for this to work. The patch is now in GitHub.